QueueMgr.py

```diff
@@ -198,24 +250,24 @@ def PutRequest(self) :
             
             #Have a special check for the GET test, which doesn't expect to get
             #any data from an actual job.
-            if request['id'] == "testgetid":
+            if request['id'] == "testgetid" or ((type(request['id']) != str) and (request['id'] < 10)):
```

Comment:
IDK, sounds 2cool4me.

I made it a 'not str' comparison in case other test commands in the future want to use some other type to distinguish themselves.  It could be switched to 'is int' if it's never an issue (or discord user ID types always stay ints).